### PR TITLE
add new filter patterns description

### PR DESCRIPTION
**Pull request by baermat**

### CODE REVIEW

File: content/en/user-guide/aws/logs/index.md [hunk 1]
```diff
@@ -91,7 +91,27 @@ awslocal cloudwatch get-metric-statistics --namespace MyNamespace \
...
+For purely JSON structured log messages, you can use JSON filter patterns to traverse the JSON object.
+Enclose your pattern in curly braces, like this: 
 {{< command >}}
 $ awslocal logs filter-log-events --log-group-name test-filter --filter-pattern "{$.foo = \"bar\"}"
-{{< / command >}}
\ No newline at end of file
+{{< / command >}}
+This returns all events whose top level "foo" key has the "bar" value.
+
+#### Regular Experession Filter Patterns:
```
Comment (steffyP) [inline content/en/user-guide/aws/logs/index.md:104]:
```suggestion
#### Regular Expression Filter Pattern
```

File: content/en/user-guide/aws/logs/index.md [hunk 2]
```diff
@@ -91,7 +91,27 @@ awslocal cloudwatch get-metric-statistics --namespace MyNamespace \
...
+You can use a simplified regex syntax for regular expression matching.
+Enclose your pattern in percentage signs like this:
+{{< command >}}
+$ awslocal logs filter-log-events --log-group-name test-filter --filter-pattern "\%[fF]oo\%"
+{{< / command >}}
+This returns all events containing "Foo" or "foo".
+For a complete set of the supported syntax, check [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html#regex-expressions)
```
Comment (steffyP) [inline content/en/user-guide/aws/logs/index.md:111]:
```suggestion
For a complete set of the supported syntax, check [the official AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html#regex-expressions).
```

File: content/en/user-guide/aws/logs/index.md [hunk 3]
```diff
@@ -91,7 +91,27 @@ awslocal cloudwatch get-metric-statistics --namespace MyNamespace \
...
+
+#### Unstructued Filter Patterns:
```
Comment (steffyP) [inline content/en/user-guide/aws/logs/index.md:113]:
```suggestion
#### Unstructured Filter Pattern
```

File: content/en/user-guide/aws/logs/index.md [hunk 4]
```diff
@@ -91,7 +91,27 @@ awslocal cloudwatch get-metric-statistics --namespace MyNamespace \
 {{< / command >}}
 
 ### Filter Log Events
-Similarly, you can use filter-pattern to filter logs, like here:
+Similarly, you can use filter-pattern to filter logs with different kinds of patterns as described by [AWS](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html).
+
+#### JSON Filter Patterns
```
Comment (steffyP) [inline content/en/user-guide/aws/logs/index.md:96]:
```suggestion
#### JSON Filter Pattern
```

**Review by steffyP — COMMENTED** [review]
Nice 🚀  Happy to see the docs updated 👍 
A found some typos though :) 

Also added a few NITs: personally I prefer having the singular form of "pattern" in the headers, and not ending the header with a colon 😅 
but up to you to decide - just keep it consistent :)

**Comment by baermat** [comment]
That's a lot of typos :grimacing: thanks!

**Review by steffyP — APPROVED** [review]

**Pull request merged**